packages/docusaurus-theme-openapi/package.json

```diff
@@ -46,6 +47,7 @@
     "docusaurus-plugin-openapi": "^0.6.4",
     "immer": "^9.0.7",
     "lodash": "^4.17.20",
+    "marked": "^4.0.0",
```

Comment:
I've gone with Marked v4.x here because Marked v5.x requires node 18 or higher.
Currently these packages declare compatibility with node >= 14.
Dropping support for older node versions is an issue for another time.

Reply:
I think it’s okay if we drop support for node 14 since its end of life

Reply:
Yeah I will update this.

Docusaurus v3 requires Node 18+ so I think the min node version for all these packages should now be >=18 as of 0.7.0 anyway.